server/features/steps/steps.py

```diff
@@ -53,7 +53,7 @@ def steip_impl_store_indexed_item_to_ctx(context, tag, index):
 def step_impl_then_get_event_file(context):
     assert_200(context.response)
     data = get_json_data(context.response)
-    url = '/upload/%s/raw' % data['filemeta']['media_id']
+    url = '/upload-raw/%s' % data['filemeta']['media_id']
```

Comment:
This was changed in superdesk-core on Apr 14 with this commit: superdesk/superdesk-core@6e0fc66